Tests/test_is.py

```diff
@@ -13,36 +13,41 @@
 #
 #####################################################################################
 
-from iptest.assert_util import *
+import unittest
 
-from System import Nullable
+from iptest import is_cli
 
-def test_object():
-    a = object()
-    b = object()
+@unittest.skipUnless(is_cli, 'IronPython specific test case')
```

Comment:
Move the skipUnless over to test_bool_nullablebool? test_object should work with CPython.